crates/fluvio-version-manager/src/common/version_directory.rs

```diff
@@ -117,6 +120,20 @@ impl VersionDirectory {
 
         Ok((manifests, active_version))
     }
+
+    pub fn check_running_processes() -> Result<()> {
```

Comment:
local bats test?

Reply:
I have been testing locally but tests hang for this scenario, will push and see how it goes on CI!

Reply:
Okay! Disowning the `run` process and adding some sleep did the trick!

```
run bash -c 'fluvio cluster start --local' &
```